config/envs/unit_test.py

```diff
@@ -2,6 +2,8 @@
 
 from config.envs.default import DefaultConfig
 
+# flake8: noqa
```

Comment:
Can we remove this? Is it possible to resolve the specific flake8 error instead?